src/slack_model.jl

```diff
@@ -5,32 +5,47 @@ export SlackModel, SlackNLSModel,
        hess_coord, hess, hprod, hprod!
 
 
-"""A model whose only inequality constraints are bounds.
+@doc raw"""A model whose only inequality constraints are bounds.
 
 Given a model, this type represents a second model in which slack variables are
 introduced so as to convert linear and nonlinear inequality constraints to
 equality constraints and bounds. More precisely, if the original model has the
 form
 
-\\\\[ \\min f(x)  \\mbox{ s. t. }  c_L \\leq c(x) \\leq c_U \\mbox{ and }
-\\ell \\leq x \\leq u, \\\\]
+```math
+\begin{align}
+&\min_x      & &f(x)\\
+&\text{s.t.} & &\begin{array}{ccccc}
+                  c_L &≤& c(x) &≤& c_U,\\
+                   l  &≤&   x  &≤&  u,
+                \end{array}
+\end{align}
+```
 
 the new model appears to the user as
 
-\\\\[ \\min f(X)  \\mbox{ s. t. }  g(X) = 0 \\mbox{ and } L \\leq X \\leq U. \\\\]
+```math
+&\min_X      & &f(X)\\
+&\text{s.t.} & &g(X) = 0,\\
+&            & &L ≤ X ≤ U.
+```
 
-The unknowns \$X = (x, s)\$ contain the original variables and slack variables
-\$s\$. The latter are such that the new model has the general form
+The unknowns ``X = (x, s)`` contain the original variables and slack variables
+``s``. The latter are such that the new model has the general form
 
-\\\\[ \\min f(x)  \\mbox{ s. t. }  c(x) - s = 0, c_L \\leq s \\leq c_U \\mbox{ and }
-\\ell \\leq x \\leq u, \\\\]
```

Comment:
I changed all `\ell` to plain `l`. Is there any reason to prefer `\ell` over `l`? I just saw some other `\ell`s in some markdown files. Maybe we should keep accordance?

Reply:
Lowercase L looks like 1, and though it's usually clear from context, we prefer `\ell`.

Reply:
I see. I changed back all the `\ell`s in the new commit.